src/components/Standings.jsx

```diff
@@ -0,0 +1,15 @@
+import { tr } from 'date-fns/locale';
+import React from 'react';
+
+function Standings({ standing }) {
+  return (
+    <tr>
+      <td className="values">{`${standing.rank}`}</td>
```

Comment:
You don't the embed the value in a string here - you can just  do this:
```
<td className="values">{standing.rank}</td>
```

Reply:
fixed thanks.